File: algorithms/F/Find N Unique Integers Sum up to Zero/Find N Unique Integers Sum up to Zero.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        if(n == 1){
            return {0};
        }else{
            vector<int> res;
            for(int i=n/2*-1;i<=n/2;i++){
                if(i == 0){
                    if(n%2 == 0){
                        continue;
                    }else{
                        res.push_back(i);
                        continue;
                    } 
                }
                res.push_back(i);
            }
         return res;   
        }
    }
};

File: scripts/algorithms/F/Find N Unique Integers Sum up to Zero/Find N Unique Integers Sum up to Zero.js
// Runtime: 50 ms (Top 86.19%) | Memory: 43.00 MB (Top 14.55%)

var sumZero = function(n) {
  var num = Math.floor(n/2); 
  var res = [];

  for(var i=1;i<=num;i++){
      res.push(i,-i)
     } 

  if(n%2!==0){
    res.push(0)
  }
  
  return res 
}
